pkg/util/ip/ip.go

```diff
@@ -32,6 +32,20 @@ const (
 	V6BitLen = 8 * net.IPv6len
 )
 
+type IPVersion int
```

Comment:
Please create an issue to replace all v1alpha2.IPv6/ v1alpha2.IPv4/ipVersion in this package.